debug.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+if [ $# -lt 2 ]; then
+	cat <<USAGE
+Usage: $0 BROWSER JENKINS [ARGS]
+
+The script runs dryrun tests first to discover trivial problems immediately.
+It can use jenkins.war from local maven repository or download it when missing.
+
+BROWSER: Value for BROWSER variable
+JENKINS: Path to the jenkins.war, Jenkins version of one of "latest", "latest-rc", "lts" and "lts-rc"
+
+Examples:
+
+# Run full suite in FF against ./jenkins.war.
+$ ./run firefox ./jenkins.war
```

Comment:
🐛 if you plan to keep a file copy, it needs to be updated to ./debug.sh . Makes sense to ensure that the file is also executable